website/content/en/status/report-2023-04-2023-06/mfsbsd-building-integration.adoc

```diff
@@ -0,0 +1,13 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile.
+The problem this project aims to address is that, since only the release versions of mfsBSD images are produced currently, they tend to get out of sync with the tools in base.
```

Comment:
```suggestion
Currently, only the release versions of mfsBSD images are produced.
This means they tend to get out of sync with the tools in base.
This project aims to address that problem.
```

Better flow by splitting it into 3 sentences.

Reply:
Very nice change! Thank you for the suggestion!